skype/Dockerfile

```diff
@@ -25,7 +25,8 @@ RUN dpkg --add-architecture i386 && \
 
 # Install Skype
 RUN curl http://download.skype.com/linux/skype-debian_4.3.0.37-1_i386.deb > /usr/src/skype.deb && \
-	dpkg --force-depends -i /usr/src/skype.deb && \
```

Comment:
can we change the image base back to jessie instead, sorry, i just feel a bit safer with that